pytentiostat/config_reader.py

```diff
@@ -56,6 +77,26 @@ def get_output_params(config_data, override_ts=None):
 
 
 def get_lsv_params(config_data):
+    """
+    Obtains the parameters for a linear sweep voltammetry experiment
+
+    Parameters
+    ----------
+    config_data : dict
+        the configuration data
```

Comment:
capitalize the here, remove mt lines, but this is super!

Reply:
Got it!